packages/ffe-grid-react/src/background-colors.js

```diff
@@ -0,0 +1,10 @@
+export default [
+    'blue-cobalt',
+    'blue-ice',
+    'blue-pale',
+    'blue-royal',
+    'green-mint',
+    'grey-cloud',
+    'purple-magenta',
+    'sand',
+];
```

Comment:
Prettier doesn't add an empty newline at EOF? :police_car:

Reply:
Doesn't look like it - but the linter should at least warn. It doesn't though. Anyhow, fixed!